### PR TITLE
Group 6

### PR DESCRIPTION
**Pull request by johanneskross**

### CODE REVIEW

File: src/main/java/Airport.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import models.MilitaryType;
+import Planes.MilitaryPlane;
+import Planes.Plane;
+
+import java.util.*;
```
Comment (kjkuhn) [inline src/main/java/Airport.java:5]: Please change the import with * to specific imports

**Review by kjkuhn — COMMENTED** [review]
first comment

File: src/main/java/Airport.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+public class Airport {
```
Comment (GustavEye) [inline src/main/java/Airport.java:7]: Add description about class. Maybe using doxygen or other generator style

File: src/main/java/Airport.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+    /**
+     * Prints transport military planes
```
Comment (GustavEye) [inline src/main/java/Airport.java:10]: Description: largest military plane?

File: src/main/java/Airport.java [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+     */
+    private void printMaxTransportMilitaryPlane() {
+        List<MilitaryPlane> transportMilitaryPlanes = new ArrayList<>();
+        Plane maxPlane = null;
+
+        // print banner
```
Comment (kjkuhn) [inline src/main/java/Airport.java:16]: Split into seperate functions

Comment (GustavEye) [inline src/main/java/Airport.java:16]: Export to separate function

File: src/main/java/Airport.java [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+        for (Plane plane : planes) {
+            if (plane instanceof MilitaryPlane) {
+                if (((MilitaryPlane) plane).getType() == MilitaryType.TRANSPORT) {
+                    if (maxPlane != null && maxPlane.getMaxLoadCapacity() < plane.getMaxLoadCapacity()) {
+                        maxPlane = plane;
+                    }
+                }
+            } //if
+            else {
```
Comment (GustavEye) [inline src/main/java/Airport.java:29]: Empty else?

File: src/main/java/Airport.java [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+import Planes.Plane;
+
+import java.util.*;
+
+public class Airport {
+
+    /**
+     * Prints transport military planes
+     */
+    private void printMaxTransportMilitaryPlane() {
```
Comment (GustavEye) [inline src/main/java/Airport.java:12]: Group public/private and functions/variables

**Pull request closed without merging**